modules/jaeger-upgrading-es.adoc

```diff
@@ -0,0 +1,61 @@
+////
+This PROCEDURE module included in the following assemblies:
+- jaeger_install/rhbjaeger-updating
+- service_mesh_install/updating-ossm
+////
+
+[id="upgrading_es5_es6_{context}"]
+= Upgrading from Elasticsearch 5 to Elasticsearch 6
+
+There is an issue with certificates when upgrading from Elasticsearch 5 to Elasticsearch 6.
```

Comment:
I think the first sentence alone doesn't provide enough information to be first. I think that if we're not going to name the issue, we shouldn't lead with it. I think the first thing should be what the customer is doing, which in this case is updating from E 5 to E6. Then, an overview of what you should do, then the general explanation.

I think it should be combined with the next paragraph.  

>When updating from Elasticsearch 5 to Elasticsearch 6, you must delete your Jaeger instance, then recreate the Jaeger instance because of an issue with certificates.

Reply:
Your version is much improved over mine.  Thanks!